website/developer-docs/releases/index.md

```diff
@@ -21,6 +21,8 @@
 
     -   Paste the list of commits since the previous release under the `Minor changes/fixes` section.
 
+        Run `make gen-changelog` and use the contents of `changelog.md`. Remove merged PRs from bumped dependencies unless they fix security issues or are otherwise notable. Remove merged PRs with the `website/` prefix.
```

Comment:
This looks like some slick automation. Is the content of the generated changelog.md automatically written to the new release notes file, or do we want to explicitly state that one needs to copy and paste the content into the ____.md file"? Currently the sentence of "Use the contents..." is a bit vague.